source/includes/azure/_security_rules.md

```diff
@@ -76,9 +76,9 @@ Attributes | &nbsp;
 curl -X GET \
   -H 'mc-api-key: your_api_key' \
   "https://cloudmc_endpoint/v1/services/azure/example/securityrules/subscriptions/subscriptionId/resourceGroups/cmc-example/providers/Microsoft.Network/networkSecurityGroups/:example-securityGroup/securityRules/example-securityRule"
```

Comment:
this curl has the `:example-securitygroup` format instead of `sample-network-security-group`